src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -76,19 +70,17 @@ function CollectionGalleryPage({ collectionId, username, queryRef }: CollectionG
       <Head>
         <title>{headTitle}</title>
       </Head>
-      <Page>
-        <StyledCollectionGalleryWrapper>
-          <StyledPositionedBackLink>
-            <ActionText onClick={handleBackClick}>← Back to Gallery</ActionText>
-          </StyledPositionedBackLink>
-          <CollectionGallery queryRef={query} />
-        </StyledCollectionGalleryWrapper>
-      </Page>
```

Comment:
deprecated this after discussing with jess.
1. this would appear beneath the new navbar, and a fix would be non-trivial
2. the component seems a bit redundant since the user can just click back on their browser, or use the breadcrumbs on the collection title